src/test/java/com/connexta/store/StoreITests.java

```diff
@@ -127,13 +125,11 @@ public AmazonS3Configuration testAmazonS3Configuration() {
     }
   }
 
-  private MockRestServiceServer indexMockRestServiceServer;
-
   @Inject private AmazonS3 amazonS3;
 
   @Inject
-  @Named("nonBufferingRestTemplate")
-  private RestTemplate nonBufferingRestTemplate;
+  @Named(IndexClientConfiguration.INDEX_WEB_CLIENT_BEAN)
+  private WebClient indexWebClient;
```

Comment:
❓ Why was this injected? I don't see it being used anywhere?

Reply:
This may not be needed.